.github/workflows/release.yml

```diff
@@ -75,8 +83,10 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build and Deploy Collection
+      - name: Deploy Collection on Ansible Galaxy
         uses: artis3n/ansible_galaxy_collection@v2
         with:
           api_key: '${{ secrets.GALAXY_API_KEY }}'
-          galaxy_version: '${{ env.NEW_VERSION }}'
+          build: false
```

Comment:
Are you sure you do not need `galaxy_version` here anymore ?

Reply:
Yep https://github.com/artis3n/ansible_galaxy_collection#galaxy_version